src/main/java/lotto/view/ResultView.java

```diff
@@ -0,0 +1,35 @@
+package lotto.view;
+
+import lotto.domain.Lotto;
+import lotto.domain.Lottos;
+import lotto.domain.WinningEnum;
+
+import java.math.BigDecimal;
+
+public class ResultView {
+    public static void purchaseCount(int money) {
+        System.out.println((money / Lotto.LOTTO_PRICE) + "개를 구매했습니다.");
+    }
```

Comment:
```suggestion
    public static void purchaseCount(Lottos lottos) {
        System.out.println(lottos.size() + "개를 구매했습니다.");
    }
```

상태를 가진 객체에게 물어봐도 좋을것 같아요 ㅎ